universal-application-tool-0.0.1/app/services/applicant/ApplicantQuestion.java

```diff
@@ -248,7 +238,12 @@ public TextQuestion() {
 
     @Override
     public boolean hasErrors() {
-      // There are no inherent requirements in a text question.
+      if (hasValue()) {
```

Comment:
Question: do we want to handle validation here or in the specific QuestionDefiniton? the latter would require passing applicant input so doing it here makes more sense IMO.

Reply:
I think this makes sense, since this is where we have both the definition _and_ the data.